Update Geyser Docs is failing

## Brief summary of issue:
https://github.com/Mudlet/Mudlet/actions/workflows/update-geyser-docs.yml

According to the action reports, the automatic job to update geyser docs started failing around July 4th of last year.


## Error output

### Remove Previous Docs
```
2026/02/13 02:07:42 dial tcp 172.64.80.1:22: i/o timeout
Error: Process completed with exit code 1.
```

## Extra information, such as the Mudlet version, operating system and ideas for how to solve:
If you go to the IP that it fails on (`172.64.80.1`), you get a cloudflare error page

<img width="1034" height="727" alt="Image" src="https://github.com/user-attachments/assets/de5a74ad-bf9a-40b9-9d1f-93bff64fc323" />

--- a/.github/workflows/update-geyser-docs.yml
+++ b/.github/workflows/update-geyser-docs.yml
@@ -34,7 +34,7 @@ jobs:
     - name: Remove previous docs
       uses: appleboy/ssh-action@master
       with:
-        host: mudlet.org
+        host: 45.33.14.76
         username: ${{secrets.UPLOAD_USERNAME}}
         key: ${{secrets.UPLOAD_PRIVATEKEY}}
         script: "rm -rf /home/${{secrets.UPLOAD_USERNAME}}/geyser/"
@@ -42,7 +42,7 @@ jobs:
     - name: Upload docs
       uses: appleboy/scp-action@master
       with:
-        host: mudlet.org
+        host: 45.33.14.76
         username: ${{secrets.UPLOAD_USERNAME}}
         key: ${{secrets.UPLOAD_PRIVATEKEY}}
         source: "${{github.workspace}}/src/mudlet-lua/mudlet-lua-doc/files/"

